### PR TITLE
Fix #72: Compiled reports working in native

### LINKED ISSUE #72: configuration for report file locations
It would be nice to have a configuration setting that gave the location of the report files in a project.  By default, this could be `src/main/resources/jasperreports` similar to where Qute stores it's templates.  Ideally, it would be `src/main/jasperreports`, and then the compiled reports (`.jasper`) would be added to the final artifact instead of the `.jrxml` files.

### PR DESCRIPTION
**Pull request by melloware**
Fix #72: Compiled reports working in native

@nderwin your new code let me create a new `CompiledReportBuildItem` and clean up a bunch of my native code for looking through the compiled only files.

let's see if this passes

### CODE REVIEW

File: deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java [hunk 1]
```diff
@@ -430,62 +423,64 @@ void watchReportFiles(BuildProducer<HotDeploymentWatchedFileBuildItem> watchedPa
...
-                Path outputFilePath = Path.of(outputTarget.getOutputDirectory().toString(), "classes",
-                        config.build().destination().get().toString());
+        if (!config.build().enable()) {
+            Log.debug("Automatic report compilation disabled");
+        }
 
-                if (!Files.exists(outputFilePath)) {
-                    Files.createDirectories(outputFilePath);
-                }
+        Log.debugf("Found %s report(s) to compile?", reportFiles.size());
```
Comment (nderwin) [inline deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java:434]: Rats, I forgot to remove that question mark.  Could you remove that?

**Review by nderwin — CHANGES_REQUESTED** [review]
I have faith this will work, first try.  :crossed_fingers:

File: deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java (hunk 1 above)
Comment (melloware) [reply deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java:434]: Yep I got it!

**Comment by melloware** [comment]
OK in Native mode tests its not finding `io.quarkiverse.jasperreports.it.AbstractJasperResource.compileReport(AbstractJasperResource.java:22)` so it can't find the compiled .jasper file in `/target/classes/.jasper` so I added more debug

**Pull request merged**